### PR TITLE
Add property based testing using hypothesis

### PR DESCRIPTION
**Pull request by micies**

### CODE REVIEW

File: parsigs/parse_sig_api.py [hunk 1]
```diff
@@ -6,6 +6,8 @@
 import re
 import logging
 from spacy import Language
+import translators as ts
+
```
Comment (royashcenazi) [inline parsigs/parse_sig_api.py:10]: @micies great work with Hypothesis! 
remove the old changes and rebase master as there were changes

**Review by royashcenazi — CHANGES_REQUESTED** [review]

**Comment by micies** [comment]
ok

**Comment by micies** [comment]
i'm change it

File: tests/test_parsig_api.py [hunk 2]
```diff
@@ -24,6 +24,13 @@ def test_parse_sig_period(self):
         result = self.sig_parser.parse(sig)[0]
         self.assertEqual(result, expected)
 
+    def test_parse_sig_text_and_regular_numbers_hebrow(self):
+        sig = "קח שתי טבליות איבופרופן 3 פעמים בכל שבוע"
```
Comment (royashcenazi) [inline tests/test_parsig_api.py:28]: this needs to be deleted :)

File: tests/test_hypothesis.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+from parsigs.parse_sig_api import SigParser, StructuredSig
+
+
+class TestParseSigApi(unittest.TestCase):
+    parser = SigParser()
+
+    @given(st.integers(min_value=1, max_value=30))
+    @settings(max_examples=30)
+    def test_parse_sig_strength(self, integer):
+        sig = f"Take 1 tablet of ibuprofen {integer}mg 3 times every day for 10 weeks"
```
Comment (royashcenazi) [inline tests/test_hypothesis.py:14]: rename this file to test_propery_parsig_api (the `test_` starter is for `pytest` to run this script when testing the project)

File: tests/test_propery_parsig_api.py [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+import unittest
+from parsigs.parse_sig_api import SigParser, StructuredSig
+
+
+class TestParseSigApi(unittest.TestCase):
+    parser = SigParser()
+
+    @given(st.integers(min_value=1, max_value=30))
+    @settings(max_examples=30)
+    def test_parse_sig_strength(self, integer):
```
Comment (royashcenazi) [inline tests/test_propery_parsig_api.py:13]: rename integer => strength

**Pull request merged**